scripts/launch/connect-to-cluster/default.nix

```diff
@@ -73,18 +73,21 @@ in pkgs.writeScript "${executable}-connect-to-${environment}" ''
   export LC_ALL=en_GB.UTF-8
   export LANG=en_GB.UTF-8
   if [ ! -d ${stateDir}/tls ]; then
-    mkdir ${stateDir}/tls/
-    ${pkgs.openssl}/bin/openssl req -x509 -newkey rsa:2048 -keyout ${stateDir}/tls/server.key -out ${stateDir}/tls/server.cert -days 3650 -nodes -subj "/CN=localhost"
+    mkdir -p ${stateDir}/tls/server && mkdir -p ${stateDir}/tls/client
+    ${iohkPkgs.cardano-sl-tools}/bin/cardano-x509-certificates   \
+      --server-out-dir ${stateDir}/tls/server                    \
+      --client-out-dir ${stateDir}/tls/client                    \
+      --configuration-key ${environments.${environment}.confKey} \
+      --configuration-file ${configFiles}/configuration.yaml
   fi
   ''}
 
-
   ${executables.${executable}}                                     \
     --configuration-file ${configFiles}/configuration.yaml         \
     --configuration-key ${environments.${environment}.confKey}     \
-    ${ ifWallet "--tlscert ${stateDir}/tls/server.cert"}           \
-    ${ ifWallet "--tlskey ${stateDir}/tls/server.key"}             \
-    ${ ifWallet "--tlsca ${stateDir}/tls/server.cert"}             \
+    ${ ifWallet "--tlscert ${stateDir}/tls/server/server.cert"}    \
+    ${ ifWallet "--tlskey ${stateDir}/tls/server/server.key"}      \
+    ${ ifWallet "--tlsca ${stateDir}/tls/server/ca.cert"}          \
     --log-config ${configFiles}/log-config-connect-to-cluster.yaml \
     --topology "${configFiles}/topology.yaml"                      \
     --logs-prefix "${stateDir}/logs"                               \
```

Comment:
Not sure about this whole file. How to actually test this :s ? 
Forgive my nix-noobness, I'll change I promise.

Reply:
This was mentioned in slack, but for anyone else wanting to test this:
`nix-build -A connectScripts.mainnetWallet -o "./launch_mainnet"`
`nix-build -A connectScripts.stagingWallet -o "./launch_staging"`

These wallets can be customized further by copying `sample-wallet-config.nix` to `custom-wallet-config.nix`, altering and rebuilding.